types/ipcheck/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Please use a file containing just `{ "extends": "tslint.json" }`. Let me know if you have trouble with any particular lint rule.

Reply:
Just the 'interface-name` rule. All interfaces here start with an `IP`, causing the rule of:
```
interface-name  interface name must not have an "I" prefix
```
to trigger needlessly. I disabled this now with a:
```typescript
// tslint:disable-next-line:interface-name
```
but I don't like it very much. If you have any better suggestions, I'm all ears.

Reply:
Just disable that rule for now then.